hack/tests/subcommand-scorecard.sh

```diff
@@ -6,18 +6,19 @@ CONFIG_PATH_V1ALPHA1=".test-osdk-scorecard-v1alpha1.yaml"
 CONFIG_PATH_DISABLE=".osdk-scorecard-disable.yaml"
 CONFIG_PATH_INVALID=".osdk-scorecard-invalid.yaml"
 CONFIG_PATH_V1ALPHA2=".osdk-scorecard-v1alpha2.yaml"
+CONFIG_PATH_BUNDLE=".osdk-scorecard-bundle.yaml"
 
 set -ex
 
+source ./hack/common.sh
```

Comment:
For a follow-on, we should move/refactor `./hack/common.sh` into `./hack/lib/`

Reply:
ok, will submit a separate PR to delete remove hack/common.sh